src/MyBrokenClass.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class MyBrokenClass
+{
+    protected string $test;
+
+    public function test(): string
+    {
+        foreach ( $_POST['menu-locations'] as $menu_id ) {
```

Comment:
:warning: **Warning**: Detected usage of a non-sanitized input variable: $_POST['menu-locations'] (*WordPress.Security.ValidatedSanitizedInput.InputNotSanitized*).